rules/windows/privilege_escalation_uac_bypass_diskcleanup_hijack.toml

```diff
@@ -11,20 +11,19 @@ stealthily execute code with elevated permissions.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License"
 name = "UAC Bypass via DiskCleanup Scheduled Task Hijack"
 risk_score = 47
 rule_id = "1dcc51f6-ba26-49e7-9ef4-2655abb2361e"
 severity = "medium"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.args:(/autoclean or /AUTOCLEAN) and
-  process.parent.name:svchost.exe and
-  not process.executable:("C:\Windows\System32\cleanmgr.exe" or "C:\Windows\SysWOW64\cleanmgr.exe")
+process where event.type in ("start", "process_started") and
+process.args:"/autoclean" and process.args:"/d" and
+not process.executable : ("C:\\Windows\\System32\\cleanmgr.exe", "C:\\Windows\\SysWOW64\\cleanmgr.exe")
```

Comment:
LGTM, I do wonder if sometime in the future we should consider wildcarding out the root drive (`C:`) in lots of our rules.  There is possibility, organizations have custom builds and these rules would not trigger.

Reply:
agree, but for performance reasons we may apply that as a bulk change once wildcard is stable.